src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days >= 7) {
+    sum = days * 40 - 50;
```

Comment:
Use variables for the main values (40, 20, 50) so that you can reuse them and give them descriptive names